examples/04-theming/06-code-block/README.md

```diff
@@ -0,0 +1,33 @@
+# Code Block Highlighting
```

Comment:
The pattern we currently use is that we have:
- max 1 or 2 paragraphs in the example as description
- an optional "try it out" line to explain how the user can see the functionality in the demo
- links to relevant docs

The idea is that the examples can really be a larger set of examples to click through and play with without too much reading - and that all documentation should be in /docs.

Could we change this? or lmk if you like a different setup

Reply:
Totally makes sense. Updated